duckduckgo/Android
# Add start-up traces internal feature

Task/Issue URL: https://app.asana.com/0/414730916066338/1201472414037474/f

### Description
Add the ability to enable/disable startup traces through internal features

### Steps to test this PR
* install internal debug/release build from this branch
* launch the app
* Open "device file explorer" in IntelliJ
* go to `data/data/com.duckduckgo.mobile.android.debug` folder
* verify there is no file with extension `.trace`
* go to "Settings" -> "Developer Settings" and ENABLE `Start-up tracing`
* kill app and re open
* Open "device file explorer" in IntelliJ
* go to `data/data/com.duckduckgo.mobile.android.debug` folder
* verify there is one file with extension`.trace` and that its size is > 0kb
* go to "Settings" -> "Developer Settings" and DISABLE `Start-up tracing`
* kill app and re open
* Open "device file explorer" in IntelliJ
* go to `data/data/com.duckduckgo.mobile.android.debug` folder
* verify there is no new file with extension `.trace`, only the previous file
